crates/bevy_pbr/src/render/mesh/mesh.wgsl

```diff
@@ -24,16 +24,43 @@ struct Vertex {
     @location(5) joint_indices: vec4<u32>,
     @location(6) joint_weights: vec4<f32>,
 #endif
+#ifdef MORPH_TARGETS
+    @builtin(vertex_index) index: u32,
+#endif
 };
 
 struct VertexOutput {
     @builtin(position) clip_position: vec4<f32>,
     #import bevy_pbr::mesh_vertex_output
 };
 
+#ifdef MORPH_TARGETS
+fn morph_vertex(vertex: Vertex) -> Vertex {
+    var vertex = vertex;
```

Comment:
This is forbidden in wgsl, wgpu accepts it but chrome validation rejects it. You can't redefine a variable with the same identifier and the same end of scope